docs/AcceptableUnicode.md

```diff
@@ -0,0 +1,92 @@
+# Acceptable Unicode characters:
+Using Unicode seems to be irresistible. However, we must ensure
+avoiding problematic Unicode usage. Here is a list of acceptable
+and forbidden Unicode characters and accents. Please note that
+we forbid the use of accents, because this can lead to visually
+ambiguous characters (see below).
+
+## Acceptable lower-case Greek letters:
+α # (alpha)
+β # (beta)
+δ # (delta)
+ϵ # (epsilon)
+γ # (gamma)
+κ # (kappa)
+λ # (lambda)
+μ # (mu)
+η # (eta)
+ω # (omega)
+π # (pi)
+ρ # (rho)
+σ # (sigma)
+θ # (theta)
+χ # (chi)
+ξ # (xi)
+ζ # (zeta)
+ϕ # (psi)
+φ # (varphi)
+
+## Acceptable upper-case Greek letters:
+Δ # (Delta)
+∑ # (Sigma)
+Γ # (Gamma)
+Ω # (Omega)
+Φ # (Phi)
```

Comment:
I find this hard to distinguish from the lowercase is some fonts.

Reply:
Yes, `Phi` and `psi` are similar.. Maybe we should remove `Phi`?

Reply:
Yes, the `\Phi` and `\psi` are very similar. I can remove `\Phi`.

Reply:
Can you give an example?

Reply:
I meant \phi and \Phi (lowercase and uppercase phi)

Reply:
Oh, okay. Regardless, I removed `\Phi` from the list for now.